e2e/test-component/scenarios/embedding-sdk/interactive-dashboard.cy.spec.tsx

```diff
@@ -71,4 +80,40 @@ describeEE("scenarios > embedding-sdk > interactive-dashboard", () => {
       cy.contains("This is a custom question layout.");
     });
   });
+
+  it("should not trigger url click behaviors when clicking on cards (metabase#51099)", () => {
```

Comment:
Can you double check this test? I tried running this test with `master` code and it still passed, but instead, the only other existing test in this file failed.

![Screenshot 2024-12-24 at 4 26 45 PM](https://github.com/user-attachments/assets/89cf793f-1674-49d7-9867-61acfd692096)

If the test works as expected it should fail on master and pass only on this PR.